[2024-05-09T10:12:33.704+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-09T10:12:33.739+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_hooks_v010.push_csv_file_data_to_postgres scheduled__2024-05-08T00:00:00+00:00 [queued]>
[2024-05-09T10:12:33.748+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_hooks_v010.push_csv_file_data_to_postgres scheduled__2024-05-08T00:00:00+00:00 [queued]>
[2024-05-09T10:12:33.748+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-05-09T10:12:33.767+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): push_csv_file_data_to_postgres> on 2024-05-08 00:00:00+00:00
[2024-05-09T10:12:33.773+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=11614) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-09T10:12:33.774+0000] {standard_task_runner.py:63} INFO - Started process 11615 to run task
[2024-05-09T10:12:33.774+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_hooks_v010', 'push_csv_file_data_to_postgres', 'scheduled__2024-05-08T00:00:00+00:00', '--job-id', '90', '--raw', '--subdir', 'DAGS_FOLDER/postgres_hooks.py', '--cfg-path', '/tmp/tmpg17f3nec']
[2024-05-09T10:12:33.775+0000] {standard_task_runner.py:91} INFO - Job 90: Subtask push_csv_file_data_to_postgres
[2024-05-09T10:12:33.818+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_with_postgres_hooks_v010.push_csv_file_data_to_postgres scheduled__2024-05-08T00:00:00+00:00 [running]> on host dab92a57e2e4
[2024-05-09T10:12:33.894+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='myself' AIRFLOW_CTX_DAG_ID='dag_with_postgres_hooks_v010' AIRFLOW_CTX_TASK_ID='push_csv_file_data_to_postgres' AIRFLOW_CTX_EXECUTION_DATE='2024-05-08T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-08T00:00:00+00:00'
[2024-05-09T10:12:33.894+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-09T10:12:33.917+0000] {base.py:84} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-05-09T10:12:33.921+0000] {logging_mixin.py:188} INFO - hello hi
[2024-05-09T10:12:33.922+0000] {logging_mixin.py:188} INFO - Error: invalid input syntax for type integer: "811-64-5910"
LINE 1: ...rs (order_id, date, product_id, quantity) VALUES ('811-64-59...
                                                             ^
[2024-05-09T10:12:33.923+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-09T10:12:33.923+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-09T10:12:33.932+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_hooks_v010, task_id=push_csv_file_data_to_postgres, run_id=scheduled__2024-05-08T00:00:00+00:00, execution_date=20240508T000000, start_date=20240509T101233, end_date=20240509T101233
[2024-05-09T10:12:33.948+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-09T10:12:33.964+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-09T10:12:33.966+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
